pkg/controller/federatedcluster/clusterclient.go

```diff
@@ -40,10 +40,13 @@ const (
 	LabelZoneRegion        = "failure-domain.beta.kubernetes.io/region"
 )
 
+// ClusterClient provides a method to compute the FederatedClusterStatus of a
```

Comment:
how about s/compute/get

Besides, the ClusterClient not only get the FederatedClusterStatus, but also get kubernetes cluster zones and regions.

Reply:
@pmorie `get` seems more logical to me too.